src/oic/oic/__init__.py

```diff
@@ -257,6 +257,32 @@ def add_token(self, resp):
     "enc": "%s_encrypted_response_enc",
 }
 
+rt2gt = {
+    'code': ['authorization_code'],
+    'id_token': ['implicit'],
+    'id_token token': ['implicit'],
+    'code id_token': ['authorization_code', 'implicit'],
+    'code token': ['authorization_code', 'implicit'],
+    'code id_token token': ['authorization_code', 'implicit']
+}
+
+
+def response_types_to_grant_types(response_types):
```

Comment:
This should be covered by tests.

Reply:
Especially the part that response_types may be in random order (so no one removes the 'sort' from the code by accident).

Reply:
One last thing then this is good to go.

The test with an unknown `response_type ` should also be present.

Reply:
Added